core/lib/state/Cargo.toml

```diff
@@ -22,6 +22,8 @@ tokio = { version = "1", features = ["rt"] }
 tracing = "0.1"
 itertools = "0.10.3"
 
+metrics = "0.21"
```

Comment:
If I didn't miss anything, looks like the new metrics use `vise` library. I guess this dependency can be removed?